build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
As far as I know that isn't possible - @MattGal is there some way to enforce ordering in Helix WorkItems?

As for your second question, yes, I've tested this locally and everything is as expected, and tests pass in Helix for Windows.